config.py

```diff
@@ -0,0 +1,81 @@
+"""ltd-keeper configuration and environment profiles."""
+
+import abc
+import os
+
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    """Configuration baseclass."""
+
+    __metaclass__ = abc.ABCMeta
+
+    SECRET_KEY = 'secret-key'
+    DEBUG = False
+    IGNORE_AUTH = False
+    PREFERRED_URL_SCHEME = 'http'
+
+    @abc.abstractclassmethod
+    def init_app(cls, app):
+        pass
+
+
+class DevelopmentConfig(Config):
+    """Local development configuration."""
+
+    DEBUG = True
+    IGNORE_AUTH = True
+    SQLALCHEMY_DATABASE_URI = os.environ.get('LTD_KEEPER_DEV_DB_URL') or \
+        'sqlite:///' + os.path.join(BASEDIR, 'ltd-keeper-dev.sqlite')
+    DEFAULT_USER = 'user'
+    DEFAULT_PASSWORD = 'pass'
+    AWS_ID = os.environ.get('LTD_KEEPER_AWS_ID')
+    AWS_SECRET = os.environ.get('LTD_KEEPER_AWS_SECRET')
+
+    @classmethod
+    def init_app(cls, app):
+        """Initialization hook called during create_app."""
+        pass
+
+
+class TestConfig(Config):
+    """Test configuration (for py.test harness)."""
+
+    SERVER_NAME = 'testing.io'
```

Comment:
I'd recommend you use use example.org or similar for SERVER_NAME. See: http://www.iana.org/domains/reserved and http://tools.ietf.org/html/rfc2606#section-2

At some point testing.io could be a real domain.